README.md

```diff
@@ -35,6 +35,9 @@ Portforwarding for nomad on port `4646` should bind to `127.0.0.1` and should al
 ### Starting a plain default box
 To get a running VM using the lastest release of this box run `vagrant box init fredrikhgrelland/hashistack` then `vagrant up`. The first command will add a file called `Vagrantfile` to your directory, and `vagrant up` will start a box based on the specifications of that file.
 
+### Starting a new project based on the hashistack
+This repository will release a new [template](template/README.md) into [fredrikhgrelland/vagrant-hashistack-template](https://github.com/fredrikhgrelland/vagrant-hashistack-template) on evry release.
+
```

Comment:
Why is the title `Starting a new project based on the hashistack`?

Reply:
Do you have a better suggestion?

Reply:
The text under doesn't have anything to do with starting a new project, does it?

Reply:
No it does not. This was a copy error.